teams/2/assets/css/informational-pages.css

```diff
@@ -9,34 +9,6 @@ body {
   font-family: helvetica;
```

Comment:
this is totally incorrect remove it, we are using a different font and is inherited from the html